### PR TITLE
[Tools] Enable PHPCS for CouchDB_Import_Instruments.php

### PR DESCRIPTION
**Pull request by laemtl**
Replaces #6000

### CODE REVIEW

**Comment by laemtl** [comment]
Replaced by #6789

**Comment by racostas** [comment]
LGTM.

File: tools/CouchDB_Import_Instruments.php [hunk 1]
```diff
@@ -78,83 +109,121 @@ function UpdateDataDicts($Instruments)
...
-        $from = "FROM 
-                        flag f 
-                        JOIN session s ON (s.ID=f.SessionID) 
-                        JOIN candidate c ON (c.CandID=s.CandID) 
-                        LEFT JOIN flag ddef ON (ddef.CommentID=CONCAT('DDE_', f.CommentID)) ";
-        $where = "WHERE 
-                        f.CommentID NOT LIKE 'DDE%' 
-                        AND f.Test_name=:inst
-                        AND s.Active='Y' AND c.Active='Y'";
+        $select = <<<'QUERY'
```
Comment (driusan) [inline tools/CouchDB_Import_Instruments.php:129]: This is a step backwards. Do not here heredocs to format SQL statements. PHP string literals allow newlines, just add line breaks.

File: tools/CouchDB_Import_Instruments.php [hunk 2]
```diff
@@ -78,83 +109,121 @@ function UpdateDataDicts($Instruments)
...
             $extraSelect = ", f.Data ";
 
             return $select . $extraSelect . $from . $where;
         }
 
         // add the SQL table to the query
         $extraSelect = ", i.* ";
-        $extraJoin = "JOIN " . $this->SQLDB->escape($tablename) . " i ON (i.CommentID=f.CommentID) ";
+        $extraJoin   = "JOIN " . $this->SQLDB->escape($tablename)
+        . " i ON(i.CommentID=f.CommentID) ";
```
Comment (driusan) [inline tools/CouchDB_Import_Instruments.php:177]: I'm surprised PHPCS allows this. It should be indented, not at the same level as the start of the statement.

**Comment by laemtl** [comment]
Blocked by #6913

File: tools/CouchDB_Import_Instruments.php [hunk 3]
```diff
@@ -78,83 +109,114 @@ function UpdateDataDicts($Instruments)
...
-        $from = "FROM 
-                        flag f 
-                        JOIN session s ON (s.ID=f.SessionID) 
-                        JOIN candidate c ON (c.CandID=s.CandID) 
-                        LEFT JOIN flag ddef ON (ddef.CommentID=CONCAT('DDE_', f.CommentID)) ";
-        $where = "WHERE 
-                        f.CommentID NOT LIKE 'DDE%' 
-                        AND f.Test_name=:inst
-                        AND s.Active='Y' AND c.Active='Y'";
+        $select = "SELECT \n
```
Comment (driusan) [inline tools/CouchDB_Import_Instruments.php:129]: This shouldn't have `\n`, there are explicit newlines already in the string.

Comment (laemtl) [reply tools/CouchDB_Import_Instruments.php]: @driusan Fixed

**Comment by driusan** [comment]
This has conflicts and needs to be rebased.

**Comment by laemtl** [comment]
@driusan Conflicts solved, but script can't be tested beacuse of #6913

**Comment by driusan** [comment]
Since this is just formatting I'd rather merge it and then have that issue fixed than have conflicts between the 2.

**Pull request merged**